examples/Template/Template.ino

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2017, Helium Systems, Inc.
+ * All Rights Reserved. See LICENCE.txt for license information
+ *
+ *
+ * Install the following libraries through Sketch->Manage Libraries:
+ *     - ArduinoJson
+ *     - Helium
+ * 
+ * This is a basic template for operating multiple sensors on one board
+ * in a procedural fashion. It includes one configuration variable which
+ * I will leave to you to define for your application. See the Config 
+ * example if you are not familiar with configuration variables.  
+ */
+
+#include "Arduino.h"
+#include "Board.h"
+#include "Helium.h"
+#include "HeliumUtil.h"
+#include "ArduinoJson.h"
+
+
+#define CHANNEL_NAME "Helium MQTT"
+#define CONFIG_VAR "channel.config_var"
+
+Helium  helium(&atom_serial);
+Channel channel(&helium);
+Config  config(&channel);
+
+int32_t config_var; //CONFIGURATION VARIABLE
+int state = 0;
+bool cycle = false; 
+int32_t reading_1;
+int32_t reading_2;
+
+enum States
+{
+    first,
+    second,
+    third,
+    fourth,
+    fifth
+};
+
+void
+update_config(bool stale)
+{
+    if (stale)
+    {
+        DBG_PRINT(F("Fetching Config - "));
+        int status = config.get(CONFIG_VAR, &config_var, 0);
+        report_status(status);
+
+        if (status == helium_status_OK)
+        {
+            DBG_PRINT(F("Updating Config - "));
+            status = config.set(CONFIG_VAR, config_var);
+            report_status(status);
+        }
+    }
+}
+
+void
+setup()
+{
+    Serial.begin(9600);
+    DBG_PRINTLN(F("Starting"));
+
+    // Begin communication with the Helium Atom The baud rate differs
+    // per supported board and is configured in Board.h
+    helium.begin(HELIUM_BAUD_RATE);
+
+    // Connect the Atom to the Helium Network
+    helium_connect(&helium);
+    // and do a channel connect
+    channel_create(&channel, CHANNEL_NAME);
+
+    // Get the initial interval
+    update_config(true);
+}
+
+
+
+void
+loop()
+{
+    StaticJsonBuffer<JSON_OBJECT_SIZE(2) + 100> jsonBuffer;
```

Comment:
Add a comment and a link to the arduinojson site for people to set their object size and overhead byte value. What you have here assumes two objects and 100 bytes overhead which is either incorrect or way to big depending on whether there are more readings than two.

Reply:
added comments for what users can and should do given their sensor data. This is the maximum size and is way bigger than needed for two sensor readings.